unitypack/export.py

```diff
@@ -0,0 +1,212 @@
+from io import BytesIO
+from .utils import BinaryReader
+
+
+class Vec2:
+	def __init__(self, x = 0, y = 0):
+		self.x = x
+		self.y = y
+
+	def read(self, buf):
+		self.x = buf.read_float()
+		self.y = buf.read_float()
+		return self
+
+
+class Vec3(Vec2):
+	def __init__(self, x = 0, y = 0, z = 0):
+		super().__init__(x, y)
+		self.z = z
+
+	def read(self, buf):
+		super().read(buf)
+		self.z = buf.read_float()
+		return self
+
+
+class Vec4(Vec3):
+	def __init__(self, x = 0, y = 0, z = 0, w = 0):
+		super().__init__(x, y, z)
+		self.w = w
+
+	def read(self, buf):
+		super().read(buf)
+		self.w = buf.read_float()
+		return self
+
+	def read_color(self, buf):
+		self.x = buf.read_ubyte()
+		self.y = buf.read_ubyte()
+		self.z = buf.read_ubyte()
+		self.w = buf.read_ubyte()
+		return self
+
+
+def get_num_streams(channels):
+	streams = []
+	# scan the channel's stream value for distinct entries
+	for c in channels:
+		if c["stream"] not in streams:
+			streams.append(c["stream"])
+
+	return len(streams)
+
+
+def extract_indicies(mesh):
+	indices = []
+	triangles = []
+	for sub in mesh.submeshes:
+		sub_indices = []
+		sub_triangles = []
+		buf = BinaryReader(BytesIO(mesh.index_buffer))
+		buf.seek(sub.first_byte)
+		for i in range(0, sub.index_count):
+			sub_indices.append(buf.read_uint16())
+		if sub.topology == 0:
+			sub_triangles.extend(sub_indices)
+		else:
+			raise NotImplementedError("(%s) topologies are not supported" % (mesh.name))
+
+		indices.append(sub_indices)
+		triangles.append(sub_triangles)
+
+	return {
```

Comment:
I don't think you should be creating a dict for the return here. you can just `return indices, triangles`